awapi/awapi_awiv_adapter/launch/awapi_awiv_adapter.launch.xml

```diff
@@ -173,6 +173,7 @@
       <param name="input_topic" value="$(var set_route)" />
       <param name="output_topic" value="$(var output_route)" />
       <param name="type" value="autoware_planning_msgs/msg/Route" />
+      <param name="durability" value="transient_local" />
```

Comment:
- Where is this parameter referred to? I couldn't find it.
- Is it better to have this parameter as string(enum) rather than bool(transient_local=true or false)?

Reply:
https://github.com/tier4/topic_tools/blob/cd96876133cc64e93b8315361f9c95ad5cbdc747/src/relay.cpp#L49-L104

Reply:
https://github.com/tier4/topic_tools/blob/tier4/main/include/topic_tools/policy_maps.hpp

Reply:
ah, I misunderstood that this is for awapi, I'm sorry. Thank you!